RebelAllianceBank/Classes/Admin.cs

```diff
@@ -3,12 +3,15 @@ namespace RebelAllianceBank.Classes
 {
     public class Admin : IUser
     {
-        public string Username { get; set; }
+        public int ID { get; set; }
+        public string PersonalNum { get; set; }
         public string Password { get; set; }
-
-        public Admin(string username, string password)
+        public string Surname { get; set; }
+        public string Lastname { get; set; }
```

Comment:
These aren't set anywhere. Maybe put them in the constructor?

Reply:
Fixed the surname and forename. Will not implement ID at the moment, this is due to how we are unsure on how to implement it.